src/app/dim-ui/useFixOverscrollBehavior.ts

```diff
@@ -16,9 +16,10 @@ import useResizeObserver from '@react-hook/resize-observer';
  * https://github.com/w3c/csswg-drafts/issues/3349#issuecomment-492721871 and
  * https://bugs.chromium.org/p/chromium/issues/detail?id=813094
  */
-export function useFixOverscrollBehavior(ref: React.RefObject<HTMLElement>) {
-  useResizeObserver(ref, (entry) => {
-    const elem = entry.target as HTMLElement;
+export function useFixOverscrollBehavior(...refs: React.RefObject<HTMLElement>[]) {
```

Comment:
I will move these to named params in an object with docs if we are happy with this approach. I didn't want to go do that refactor until we are sure of the solution, as I am still undecided how much I like it.

Reply:
Yeah I'm not sure either. The big question is whether this still solves the original problem, which is to allow for overscroll within a sheet without it scrolling the background.

Reply:
Yeah so on my windows PC where I reproduced the issue, using the filter help, I can conform that I can scroll the entirety of the sheet and we never see the inventory screen scroll either during or when hitting the ends. When I get a chance I will test it on mac and on my iphone to ensure touch is behaving as expected.

Reply:
Actually, what is expected behaviour with a background scrolling element? Because I see this on both prod and this branch.

![Untitled video - Made with Clipchamp](https://github.com/DestinyItemManager/DIM/assets/7344652/4236f384-aaa6-4b26-9ed8-e9101ee4c934)

Reply:
The goal is that attempting to scroll on the sheet contents should not chain to scrolling the body. This is mostly targeted at touch scrolling, but it seems like my fix doesn't even work there, now. So who knows. Maybe we should go back to body-scroll-lock libraries, if there's still a maintained one.

Reply:
Let me have a play around to see if I can get something that works everywhere. Mobile is the most important bit I think due to how disruptive this can be to the touch experience but desktop is also a nice to have.

Reply:
Hmmm this does in fact break mobile as i can now the background behind the filter help when there is no help results.

I did a bit of a dig around and the state of overscroll behaviour and overscroll chaining is a mess and has been for a while.